### ISSUE TITLE
default schema for task is not oracle friendly

### ISSUE THREAD

**Issue by paweljasinski** [codingjoe/joeflow#7]
makemigrations against oracle as db backend creates the following warnings
```
(venv) λ python manage.py makemigrations
System check identified some issues:

WARNINGS:
joeflow.Task.name: (fields.W162) Oracle does not support a database index on NCLOB columns.
        HINT: An index won't be created. Silence this warning if you don't care about it.
joeflow.Task.status: (fields.W162) Oracle does not support a database index on NCLOB columns.
        HINT: An index won't be created. Silence this warning if you don't care about it.
joeflow.Task.type: (fields.W162) Oracle does not support a database index on NCLOB columns.
        HINT: An index won't be created. Silence this warning if you don't care about it.
```
Would it make more sense to use CharFIeld instead of TextField for the 3 fields in question?
I'd prefer limited size fields with index than unlimited size fields without index.

**Comment by codingjoe** [comment]
Hi @paweljasinski thanks for reaching out. Wow, I wasn't aware people were already trying to use joeflow, considering that I didn't do much advertisement about it.

Anyhow, this package is still in an early stage, so I am happy to make those adjustments. But first, let me explain why it is a `TextField` with an index. This is mainly because PostgreSQL doesn't care ;)
To be fair, I am struggling to add Oracle support. I just recently implemented a bunch of features in Django itself and Oracle support was a nightmare.

In other words if you are willing to contribute changes, to make joeflow Oracle compatible, I won't say no.

Best,
Joe

**Comment by paweljasinski** [comment]
I suspect the fact that it is possible, doesn't mean it is a good idea to use it. There is a reason, most db schemas try to stay away from text columns of unrestricted size unless absolutely necessary.
How about other dbs? Is it possible in mssql or MySQL? Any performance consequences?
AFAIK the fields in question can have a reasonable, arbitrary maximum size.

**Comment by codingjoe** [comment]
I am just used to text fields, because I am mainly using PostgreSQL for my projects. I'd really welcome a patch. Feel free to set a length you find reasonable.
